packages/@webex/plugin-meetings/src/locus-info/selfUtils.ts

```diff
@@ -270,6 +271,9 @@ SelfUtils.getRemoteMuted = (self: any) => {
 
 SelfUtils.getLocalAudioUnmuteRequested = (self) => !!self?.controls?.audio?.requestedToUnmute;
 
+SelfUtils.getLocalAudioUnmuteRequestedTimeStamp = (self) =>
+  self?.controls?.audio?.lastModifiedRequestedToUnmute || '0';
```

Comment:
why `'0'` vs `0`

Reply:
changed it now that I am using Date.Parser